ui/DesignSystem/Component/Sidebar.svelte

```diff
@@ -1,113 +1,180 @@
+    fill: var(--color-purple);
   }
 
-  div :global(li:hover svg) {
+  .item.active :global(svg) {
     fill: var(--color-purple);
   }
 
-  div :global(li:hover .show-on-hover) {
+  .divider {
+    width: 36px;
+    height: 17px;
+  }
+
+  .line {
+    height: 1px;
+    width: 100%;
+    background-color: var(--color-gray);
+  }
+
+  .item:hover .tooltip {
     opacity: 1;
-    display: inline-block;
+    transition: 0.3s;
+  }
+
+  .tooltip {
+    user-select: none;
+    background-color: var(--color-white);
+    border: 1px solid var(--color-lightgray);
+    color: var(--color-darkgray);
+    text-align: center;
+    border-radius: 2px;
+    padding: 4px 10px 6px 8px;
+    box-shadow: 0px 4px 8px rgba(51, 51, 51, 0.08);
+
+    position: absolute;
+    opacity: 0;
+    top: 2px;
+    left: 48px;
+    pointer-events: none;
+  }
+
+  .tooltip:after,
+  .tooltip:before {
+    right: 100%;
+    top: 50%;
+    border: solid transparent;
+    content: "";
+    height: 0;
+    width: 0;
+    position: absolute;
+  }
+
+  .tooltip:after {
+    border-right-color: var(--color-white);
+    border-width: 6px;
+    margin-top: -6px;
+  }
+
+  .tooltip:before {
+    border-right-color: var(--color-lightgray);
+    border-width: 7px;
+    margin-top: -7px;
+  }
+
+  a {
+    display: flex;
+  }
+
+  .user-avatar {
+    width: 36px;
+    height: 36px;
+    border-radius: 18px;
   }
 </style>
 
-<div data-cy="sidebar">
-  <nav>
-    <ul>
-      <li>
-        <Item
-          icon={Icon.Search}
-          title="Search"
-          dataCy="search"
-          href={path.search()}
-          active={path.active(path.search(), $location)} />
-      </li>
-      <li>
-        <Item
-          icon={Icon.Feed}
-          title="Feed"
-          dataCy="feed"
-          href={path.feed()}
-          active={path.active(path.feed(), $location)} />
-      </li>
-      <li>
-        <Item
-          icon={Icon.Projects}
-          title="Projects"
-          dataCy="projects"
-          href={path.projects()}
-          active={path.active(path.projects(), $location)} />
-      </li>
-    </ul>
-  </nav>
-
-  <nav>
-    <ul>
-      <li>
-        <Item
-          icon={Icon.Plus}
-          title="Create new project"
-          dataCy="new-project"
-          href={path.createProject()}
-          active={path.active(path.createProject(), $location)} />
-      </li>
-      <li>
-        <Item
-          icon={Icon.Fund}
-          title="Fund"
-          dataCy="wallet"
-          href={path.wallet()}
-          active={path.active(path.wallet(), $location)} />
-      </li>
-      <li>
-        <Avatar
-          image="https://avatars2.githubusercontent.com/u/2326909?s=400&v=4"
-          title="Profile"
-          dataCy="profile"
-          href={path.profile()}
-          active={path.active(path.profile(), $location)} />
-      </li>
-    </ul>
-  </nav>
+<div class="wrapper" data-cy="sidebar">
+  <ul>
+    <li
+      class="item"
```

Comment:
Minor thing: like this, the cursor-change and link-action is restricted to only the icon, but the tooltip and grey bar already appear on the `li.item`. So they appear although I can't click on the sidebar item.